apps/vendor-portal/src/modules/orders/pages/orders-edit.vue

```diff
@@ -315,6 +315,49 @@ export default defineComponent({
       order,
       items: computed(() => order.value?.items),
       loading,
+      shippingInfo: computed(
```

Comment:
We should move this calculation into useOrder composable to make component more clear.